packages/openactive-integration-tests/test/helpers/logger.js

```diff
@@ -166,6 +166,20 @@ class ReporterLogger extends BaseLogger {
     return this.testName;
   }
 
+  get activeSuites() {
```

Comment:
So is this to limit to suites that were actually run (hence sourcing them from logs) as opposed to all the tests suites registered in the spec files?

Reply:
activeSuites - the idea is to only list suites with `it` blocks in, the suites array even contains suites that only contain describe() blocks (ended up with loads of section headers with no content)